doc/release-notes-0.12.md

```diff
@@ -0,0 +1,144 @@
+This page discusses the changes that you need to be aware of when migrating your Ziti deployment from version 0.11.x to version 0.12.x
+
+# Theme
+Ziti 0.12 includes the following: 
+ 
+ * Terminators have been extracted from services
+     * Terminators configurate where a service terminates. Previously each service had exactly one terminator. Now services can have 0 to N terminators.
```

Comment:
Terminators _configure_? (instead of _configurate_)

Reply:
Fixed :)